src/entry-helpers.ts

```diff
@@ -39,7 +49,12 @@ export const getConfigFromPage = (page: string) => {
   return Object.fromEntries(entries);
 };
 
-export const pushBullets = async (bullets: string[]) => {
+export const pushBullets = async (bullets: string[], title: string = "") => {
+  // There is an issue with periods which happens only for the first bullet. 
```

Comment:
woah does this not happen for subsequent bullets?

Reply:
Yup 🤷  This is a weird one. Which is why the hack is to introduce a title bullet and deliberately append a period to the title, so that the actual bullets aren't affected.